system/Database/BaseConnection.php

```diff
@@ -339,6 +340,13 @@ abstract class BaseConnection implements ConnectionInterface
      */
     protected $queryClass = Query::class;
 
+    /**
+     * Use MYSQLI_OPT_INT_AND_FLOAT_NATIVE
+     *
+     * @var bool
+     */
+    protected $DBTypeNative = false;
```

Comment:
This should be placed in the MySQLi driver, since it is related to this driver exclusively.

```suggestion
    protected bool $strictType = false;
```

Reply:
Yes, please move the property to MySQLi driver.

Reply:
Okay!